client-library/src/docker/DockerFile

```diff
@@ -0,0 +1,118 @@
+FROM mcr.microsoft.com/mirror/docker/library/ubuntu:18.04

```

Comment:
should we pull the image from some public repo?

Reply:
No, this is recommend as public repo throttles.

Reply:
yes this image is msft internal only. I will change it to use public repo ubuntu image.